modules/ding_provider/connie_search/includes/connie_search.search.inc

```diff
@@ -0,0 +1,266 @@
+<?php
+/**
+ * @file
+ * Implements handling of loans through OpenRuth.
+ */
+
+use Connie\Search\ConnieSearchResult;
+use Connie\Search\ConnieTingObject;
+use Connie\Search\ConnieTingObjectCollection;
+use Ting\Search\TingSearchCommonFields;
+use Ting\Search\TingSearchRequest;
+use Ting\Search\TingSearchSort;
+
+/**
+ * Get a list of material types from the Well.
+ *
+ * @return string[]
+ *   List of material names.
+ */
+function connie_search_search_material_types() {
+  return ['material_type_1', 'material_type_2'];
+}
+
+/**
+ * Get a list of search sources.
+ *
+ * @return string[]
+ *   List of search sources.
+ */
+function connie_search_search_sources() {
+  return ['source A', 'source B'];
+}
+
+/**
+ * Load objects from Open Search.
+ *
+ * @param string[] $ids
+ *   Array of ids to load from Open Search.
+ *
+ * @return Ting\TingObjectInterface[]
+ *   An array of loaded objects.
+ */
+function connie_search_search_object_load(array $ids) {
+  return [new ConnieTingObject()];
```

Comment:
:+1: I will update it to return an array of Ting objects where the keys match the incoming ids.

Note that it will be copies of the same dummy object.

Reply:
👍